hub/hub_nmdc.go

```diff
@@ -229,6 +229,38 @@ func (h *Hub) nmdcAccept(peer *nmdcPeer) error {
 	if err != nil {
 		return err
 	}
+
+	registeredUser := true
```

Comment:
```golang
isRegistered, err := h.isRegisteredUser(peer.Name())
```
And let `isRegisteredUser` return `true` only if the name has some special word in it, for example `_reg`. This way we can test it with and without the password.